bin/rector.php

```diff
@@ -34,7 +34,9 @@
 $extractedPhpstanAutoload = __DIR__ . '/../vendor/phpstan/phpstan-extracted/vendor/autoload.php';
 if (file_exists($extractedPhpstanAutoload)) {
     require_once $extractedPhpstanAutoload;
-} elseif (should_include_preload()) {
+}
+
+if (file_exists(__DIR__ . '/../preload.php')) {
```

Comment:
this is ensure `preload.php` always used to avoid error like in :

https://github.com/rectorphp/rector-src/pull/904/checks?check_run_id=3721183910#step:6:8